plop-templates/@react-types/src/index.d.ts.hbs

```diff
@@ -13,9 +13,15 @@
 import {DOMProps, StyleProps} from '@react-types/shared';
 
 export interface {{componentName}}Props {
-  
+  value?: any,
```

Comment:
why include these?

Reply:
Since the useState template includes a useControlledState call that uses `value`, `defaultValue`, and `onChange`, I decided to add these to the props just to reduce the lint errors someone would encounter right off the bat

Reply:
oo, good call